WalletWasabi.Fluent/TreeDataGrid/TreeDataGridPrivacyTextCell.cs

```diff
@@ -1,28 +1,28 @@
-using System.Collections.Generic;
+using System.Reactive.Disposables;
 using System.Reactive.Linq;
 using Avalonia;
 using Avalonia.Controls;
 using Avalonia.Controls.Models.TreeDataGrid;
 using Avalonia.Controls.Primitives;
 using Avalonia.Media;
 using ReactiveUI;
+using WalletWasabi.Fluent.Extensions;
 
 namespace WalletWasabi.Fluent.TreeDataGrid;
 
 internal class TreeDataGridPrivacyTextCell : TreeDataGridCell
 {
-	private static readonly List<TreeDataGridPrivacyTextCell> Realized = new();
-	private static IDisposable? Subscription;
-	private static bool IsContentVisible = true;
-	private string? _value;
+	private IDisposable _subscription = Disposable.Empty;
 	private FormattedText? _formattedText;
+	private bool _isContentVisible = true;
 	private int _numberOfPrivacyChars;
+	private string? _value;
 
-	public string? Text => IsContentVisible ? _value : new string('#', _value is not null ? _numberOfPrivacyChars : 0);
+	private string? Text => _isContentVisible ? _value : new string('#', _value is not null ? _numberOfPrivacyChars : 0);
```

Comment:
It was missing before your PR but `TextHelpers.GetPrivacyMask` could be used here to be consistent.